version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.3.0"
+version in ThisBuild := "0.3.0-SNAPSHOT"
```

Comment:
We're trying to publish again 0.3.0 with sbt-release plugin. To accomplish it, we have to reboot version.sbt with latest snapshot published.